engine/class_modules/warlock/sc_warlock.hpp

```diff
@@ -508,6 +511,9 @@ struct warlock_t : public player_t
     gain_t* baleful_invocation;  // BFA - Azerite
 
     gain_t* memory_of_lucid_dreams;  // BFA - Essence
+
+    // SL
+    gain_t* decimating_bolt;
```

Comment:
The gain here is not initialized and I can not see it used anywhere. Or is it supposed to go into the decimating_bolt_t::impact function?

Reply:
Forgot to remove it

Reply:
Forgot to remove it

Reply:
It just wasn't removed after testing, thanks